pkg/trace/agent/sampler.go

```diff
@@ -114,7 +114,7 @@ func (s *Sampler) logStats() {
 
 			switch state := state.(type) {
 			case sampler.InternalState:
-				log.Debugf("%s: inTPS: %f, outTPS: %f, maxTPS: %f, offset: %f, slope: %f, cardinality: %d",
+				log.Tracef("%s: inTPS: %f, outTPS: %f, maxTPS: %f, offset: %f, slope: %f, cardinality: %d",
```

Comment:
@AlexJF this is fine, right?

Reply:
Note that there's a debug log above with more generic info
`log.Debugf("%s: flushed %d sampled traces out of %d", engineType, keptTraceCount, totalTraceCount)`

Reply:
I guess this should be fine. I'm not aware of anyone actually making use of this.